src/App.jsx

```diff
@@ -14,33 +16,91 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const GoodsList = ({ goods }) => (
+  <>
+    {goods.map(good => (
+      <li data-cy="Good">{good}</li>
+    ))}
+  </>
+);
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+export const App = () => {
+  const [sortField, setSortField] = useState('');
+  const [isReversed, setIsReversed] = useState(false);
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  let VisibleGoods = [...goodsFromServer].sort((good1, good2) => {
+    switch (sortField) {
+      case 'alphabet':
+        return good1.localeCompare(good2);
+      case 'length':
+        return good1.length - good2.length;
+      case 'reset':
```

Comment:
Move these words 'alphabet', 'length', 'reset' to the enum and use it everywhere